embassy-stm32/src/rcc/h.rs

```diff
@@ -168,7 +168,12 @@ impl Default for Config {
             apb4_pre: APBPrescaler::DIV1,
 
             per_clock_source: PerClockSource::HSI,
-            adc_clock_source: AdcClockSource::from_bits(0), // PLL2_P on H7, HCLK on H5
+
```

Comment:
could you use concrete enum names, like `AdcClockSource::HCLK`? the `from_bits` was to have one line of code that would work on both H5 and H7, but if there's `cfg`s there's no point.

Reply:
**Sure!** I was hoping to test this on H5 to see if they could share the same (2) value, but...
...I can't compile ADC code on H5 to test this. I get this for both the stm32h503rb and the stm32h563zi: 
```
12 | use embassy_stm32::adc::Adc;
   |                    ^^^ could not find `adc` in `embassy_stm32`
```
I haven't given up completely, but I don't see what I'm doing wrong when the H7 compiles fine and I've adjusted Cargo.toml.
```
embassy-stm32 = { version = "*", path = "../embassy-stm32", features = ["nightly", "defmt", "stm32h503rb", "time-driver-any", "exti", "memory-x", "unstable-pac", "unstable-traits"] }

```
and .cargo/config.toml
```
[target.thumbv8m.main-none-eabihf]
runner = 'probe-rs run --chip STM32H503RBTx'

[build]
target = "thumbv8m.main-none-eabihf"
```

Reply:
ADC is not implemented for H5, it's normal it fails.

Reply:
Done